src/hardware/mixer.cpp

```diff
@@ -2377,7 +2377,7 @@ class MIXER final : public Program {
 	void AddMessages()
 	{
 		MSG_Add("SHELL_CMD_MIXER_HELP_LONG",
-		        "Displays or changes the sound mixer settings.\n"
+		        "\nDisplays or changes the sound mixer settings.\n"
```

Comment:
Do not start help with new line, this breaks description in `help /all` command.

Reply:
Ok.